test/models/test_property.rb

```diff
@@ -115,7 +128,44 @@ def test_missing_attributes
     prop.address = nil
 
     assert !prop.valid?
-    assert_equal 'Missing required attributes: propertyID, address', prop.missing_attributes
+    expected_errors = {
+      propertyID: 'is missing',
+      address: 'is missing'
+    }
+    assert_equal expected_errors, prop.error_messages
+  end
+
```

Comment:
Would it be worth adding a few more tests around error cases to make the testing more robust?